executor/newdeploy/newdeploy.go

```diff
@@ -203,8 +203,12 @@ func (deploy *NewDeploy) getDeploymentSpec(fn *crd.Function, env *crd.Environmen
 
 	deployment := &v1beta1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels: deployLabels,
 			Name:   deployName,
+			Labels: deployLabels,
+			Annotations: map[string]string{
+				"env-name": env.Metadata.Name,
```

Comment:
we have "envName" defined as a constant somewhere?  can we either use that or define a new constant